enterprise/server/ociregistry/ociregistry.go

```diff
@@ -0,0 +1,305 @@
+package ociregistry
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"io"
+	"net/http"
+
+	"regexp"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/buildbuddy-io/buildbuddy/server/environment"
+	"github.com/buildbuddy-io/buildbuddy/server/metrics"
+	"github.com/buildbuddy-io/buildbuddy/server/util/log"
+	"github.com/buildbuddy-io/buildbuddy/server/util/prefix"
+	"github.com/buildbuddy-io/buildbuddy/server/util/status"
+	"github.com/google/go-containerregistry/pkg/name"
+	v1 "github.com/google/go-containerregistry/pkg/v1"
```

Comment:
Dumb question but is this necessary? Asking b/c it's the same as the path.

Anyway if we have to keep it we should probably move it down with other named/aliased imports in a section below. (repb is an example)

Reply:
Glad to know it's not just me! I'm using [Zed](https://zed.dev/). My hunch is it's running `gofmt` on save, maybe there's a knob to tweak. Or maybe this is The One True Way.

Reply:
There's a `goimports` tool that we could run in checkstyle.sh that organizes imports (and also insists on adding this explicit `v1` alias). ~5 files would be changed across the codebase. My attempt to add it as a binary alongside `gofmt` failed.